src/gen_module.sh

```diff
@@ -49,16 +49,30 @@ if [ ${python_version[0]} -lt 3 ] || [[ "${python_version[0]}" -eq 3 && "${pytho
     exit 1
 fi
 
+function get_remote_git_ref() {
+    local ref=$1
+
+    # 1. Try asking remote for ref. (git ls-remote)
+    # 2. If multiple are found, take first one (head -n 1)
+    # 3. Only save hash (cut -f1)
+    local remote_ref="$(git ls-remote origin $ref | head -n 1 | cut -f1)"
```

Comment:
Question: Is there any cases which ls-remote returns more than 2 lines?

Reply:
@nutti yes. In cases where a tag and a branch have the same name.

Reply:
Thanks! I got it.